src/xunit.performance.nuspec

```diff
@@ -54,9 +54,7 @@
         <dependency id="System.Text.Encoding" version="4.0.10" />
         <dependency id="System.Threading" version="4.0.10" />
         <dependency id="System.Threading.Tasks" version="4.0.10" />
-        <dependency id="xunit.abstractions" version="2.0.0" />
-        <dependency id="xunit.extensibility.core" version="2.1.0" />
-        <dependency id="xunit.extensibility.execution" version="2.1.0" />
+        <dependency id="Microsoft.DotNet.BuildTools.TestSuite" version="1.0.0-prerelease-508-01" />
```

Comment:
Thanks.  It occurs to me that we also need to make sure that this doesn't break the perf tooling in the CLI repo - things are slightly different there at this time.

Can you work with @dpodder to make sure that the CLI perf testing scripts work with these changes?